playground/terraform/infrastructure/cloudbuild-manual-setup/02.builders/secret.tf

```diff
@@ -16,7 +20,7 @@ resource "google_secret_manager_secret" "secret_gh_pat_cloudbuild" {
 
 resource "google_secret_manager_secret_version" "secret_webhook_cloudbuild_trigger_cicd_data" {
   secret = google_secret_manager_secret.secret_webhook_cloudbuild_trigger_cicd.id
-  secret_data = var.data_for_cicd_webhook_secret
+  secret_data = random_string.data_for_cicd_webhook_secret.result
```

Comment:
This also should be pre-configured by user in GitHub WebHooks. Request the parameter,